### DISCUSSION TITLE
[ FF7 Steam ] Read Save of 2 folders at the same time, and Save are deleted

### DISCUSSION THREAD

**Discussion by ffsaga** [julianxhokaxhiu/FFNx#727] [Q&A] [answered]
Read Save of 2 folders at the same time:
After starting the game, a save folder will be forced to be created in the game folder.
If the game folder\save already exists and there are save files in it, the game will read the save files in the game folder\save.
If there are save in %USERPROFILE%\Doc..\Squ..\FI..\user_<user-id>, the game will also read the save here.
If the game folder\save has save 1.2.3, and %USERPROFILE%\Doc..\Squ..\FI..\user_<user-id> has save 4.5.6, after entering the game, you will find that there is save 1.2.3.4.5.6
But if %USERPROFILE%\Doc..\Squ..\FI..\user_<user-id> also has save 1.2.3, after entering the game, you will find the content of save 1.2.3, which is from the game folder\save

Save are deleted:
After starting FF7_Launcher
Delete metadata.xml in %USERPROFILE%\Do..\Sq..\FI..\user_<user-id>\
Start the game and load any save
Then save to any slot
The game will create an empty metadata.xml with only <?xml version="1.0"?>
( If the FFNx driver is not used, the game will create a complete metadata.xml )
If you close the game, then close FF7_Launcher
Restart FF7_Launcher
The save in %USERPROFILE%\Do..\Sq..\FI..\user_<user-id>\ will be deleted.

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hello @ffsaga not sure what are you trying to accomplish here, but let me try to unpack.

---

> Read Save of 2 folders at the same time

This was added as a backup in case people run `ff7_en.exe` next to `ff7.exe` converted via 7th. Since now 7th is going to use the native edition this won't be necessary anymore. However, it is a known behavior that FFNx can read from the `save` folder next to `ff7_en.exe` but write to where Steam expects to find them. This behavior won't change, and is not a bug.

> Save are deleted

If you delete the `metadata.xml` manually by hand, it is a known issue. See #684
Until the patch comes, all I can suggest is since you use FFNx, to use the custom launcher I built which will then completely remove the issue of saves as you won't use the one by SE anymore ( which is the one that deletes the saves ). Here you can find it https://github.com/julianxhokaxhiu/FF78Launcher

---

Hope this helps.
